src/scaladoc/scala/tools/nsc/doc/DocFactory.scala

```diff
@@ -87,12 +87,12 @@ class DocFactory(val reporter: Reporter, val settings: doc.Settings) { processor
 
     modelFactory.makeModel match {
       case Some(madeModel) =>
-        if (!settings.scaladocQuietRun)
-          println("model contains " + modelFactory.templatesCount + " documentable templates")
+        if (settings.verbose)
+          reporter.echo("model contains " + modelFactory.templatesCount + " documentable templates")
         Some(madeModel)
       case None =>
-        if (!settings.scaladocQuietRun)
-          println("no documentable class found in compilation units")
+        if (settings.verbose)
+          reporter.echo(null, "no documentable class found in compilation units")
```

Comment:
What's the `null`? Does it do something different than `NoPosition`?

Reply:
We need `NullPosition`.

Reply:
I copied `reporter.echo(null, ...)` from a few lines below, but there's actually an overload of echo that doesn't take a position it seems.

Reply:
We need an overload of `echo` that takes `NullPosition`.